lib/services/azuresqldb/cmd-provision.js

```diff
@@ -166,8 +166,12 @@ var sqldbProvision = function (log, params) {
                     } else if (firewallRules !== null) {
                         var errorMessage = util.format('You specify the firewall rules for the server but you do not have the privilege to operate the server.');
                         callback(Error(errorMessage));
-                    } else {
+                    } else if (result.statusCode === HttpStatus.OK) {
+                        fullyQualifiedDomainName = JSON.parse(result.body).properties.fullyQualifiedDomainName;
                         callback(null);
+                    } else {
+                        var errorMessage = util.format('Unexpected error: %j', result);
+                        callback(Error(errorMessage));
                     }
                 },
                 function (callback) {   // see if db exists (in the case that sql server was there already)
```

Comment:
As we discuss before, the integration test case for the existing sql server can't be added easily. We can't change the privilege whether allow to create server or not. Though we have such a case, we still need to modify some env variables and the test cases to use. I don't think it is good for long term. We need to figure out a better way to do this.